src/streaming/protection/servers/Primetime.js

```diff
@@ -29,17 +29,32 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  */
 
-/**
- * Adobe Access DRM
- *
- * @class
- * @implements MediaPlayer.dependencies.protection.KeySystem
- */
-MediaPlayer.dependencies.protection.KeySystem_Access = function() {
+MediaPlayer.dependencies.protection.servers.Primetime = function() {
     "use strict";
-};
 
-MediaPlayer.dependencies.protection.KeySystem_Access.prototype = {
-    constructor: MediaPlayer.dependencies.protection.KeySystem_Access
+    return {
+
+        getServerURLFromMessage: function(url, message, messageType) {
+            if (messageType === "individualization-request") {
+                url = 'http://individualization.adobe.com/flashaccess/i15n/v5';
```

Comment:
@bwidtmann Is this the only URL that can ever be used for Primetime individualization requests?  Or is it possible for an Adobe customer to deploy their own implementation of this server (possibly using an Adobe SDK of some sort)?  If the former, then we can leave this here.  If the latter, we need to move this information into the ProtectionData class and allow it to be specified programmatically by any application using the dash.js player library.